app/static/voice.js

```diff
@@ -0,0 +1,82 @@
+function listVoices(synth, voiceSelect) {
+	const voices = synth.getVoices();
```

Comment:
Be consistent with the amount of spaces in indentations.

Make sure you have no tabs, and 2 spaces for each intendation.

Reply:
I hope now the indentations are correct :)